controller/QtiCreator.php

```diff
@@ -348,4 +363,66 @@ private function validateXmlInput(string $xml)
         }
         \tao_helpers_Xml::getSimpleXml($xml);
     }
+
+    /**
+     * Download sample template for tabular import
+     *
+     * @param string
```

Comment:
Please remove redundant phpdoc blocks
```suggestion
     * @param string
```

Reply:
Suggested changes applied